src/server/auth/server/testing/oidc_test.go

```diff
@@ -57,7 +60,7 @@ func TestOIDCAuthCodeFlow(t *testing.T) {
 	// to the login page. The params have the session state. POST our hard-coded
 	// credentials to the login page.
 	vals := make(url.Values)
-	vals.Add("login", "admin@example.com")
+	vals.Add("login", "admin")
```

Comment:
The new username for the mockPassword connector is slightly different from our old, fixed email for no particular reason.